src/legacy/core_plugins/kibana/public/visualize/listing/visualize_listing_table.js

```diff
@@ -103,10 +103,31 @@ class VisualizeListingTableUi extends Component {
         render: (field, record) =>  (
           <span>
             {this.renderItemTypeIcon(record)}
-            {record.type.title}
+            {record.typeTitle}
             {this.getExperimentalBadge(record)}
           </span>
         )
+      },
+      {
```

Comment:
I might be mistaken, butI thought we talked offline about this and that it should be possible to move this into the table list view because it just has a single other consumer. Then we don't have to specify the "actions" column in two places, here and in `src/legacy/core_plugins/kibana/public/table_list_view/table_list_view.js`

Reply:
Yeah. It's not 1:1, though, since this allows per-item disabling. But I agree it should be consolidated. I'll update the master PR.